### PR TITLE
Home screen

### PR DESCRIPTION
**Pull request by AP7557**
Added Icons and temporary services list, as well as a header

### CODE REVIEW

**Review by ngp23 — APPROVED** [review]
Looks good to me and it works fine for me

File: components/Services.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	{
+		Service: "Roofing & Guttur",
+		Img: "https://verycozyhome.com/wp-content/uploads/2020/07/importance-of-lawn-care.jpg",
+	},
+	{
+		Service: "Moving",
+		Img: "https://verycozyhome.com/wp-content/uploads/2020/07/importance-of-lawn-care.jpg",
+	},
+    {
+		Service: "Moving",
```
Comment (juhichaudhari) [inline components/Services.tsx:35]: remove the extra

**Review by juhichaudhari — DISMISSED** [review]

File: components/Services.tsx (hunk 1 above)
Comment (pxrth9) [reply components/Services.tsx:35]: this!

**Comment by AP7557** [comment]
Re-did the ui, and made a search functionality

**Comment by pxrth9** [comment]
image

**Comment by pxrth9** [comment]
when selecting search, can you make it so that it doesn't get the black box surrounded by it, and add a search icon to the far-left or far-right depending on the thing that juhi sent?

**Comment by AP7557** [comment]
> when selecting search, can you make it so that it doesn't get the black box surrounded by it, and add a search icon to the far-left or far-right depending on the thing that juhi sent?

how did u get it were you on phone cuz i dont get it when i click

**Comment by pxrth9** [comment]
LGTM 🚀 Great Job!

**Review by pxrth9 — APPROVED** [review]

**Pull request merged**